lib/java/com/google/android/material/card/MaterialCardView.java

```diff
@@ -632,4 +647,20 @@ private void forceRippleRedrawIfNeeded() {
       cardViewHelper.forceRippleRedraw();
     }
   }
+
+
+
+  @Override
```

Comment:
Can't you just use existing setShapeAppearanceModel()? Why do you need to override onDraw() and onSizeChanged()?

Reply:
If i only use shape appearance model then the content of the card will be overflow on corners.